### DISCUSSION TITLE
Intermediate optimization level?

### DISCUSSION THREAD

**Discussion by SimonSapin** [servo/servo#9904] [General] [answered]
Release builds take forever to make, and debug build take forever to run. Browser.html in particular is almost unusable (at least for me) in debug mode: wait multiple seconds or so between hitting the keyboard and a letter showing up in the URL bar, multiple seconds to bring up the "tab" list with only one intermediate frame of the animation showing up…

Can we do something in-between? Maybe `-O1`, or only optimizing some crates?

**Comment by nox** [comment]
@SimonSapin Did we ever take a decision on this?

**Comment by SimonSapin** [comment, marked as answer]
This is five years old, and I don’t feel the need to do anything here anymore. Release (with debug assertions, to match WPT CI) is my default config in `.servobuild`, it takes about the same time to build as debug on my desktop machine. I haven’t tried recently on a laptop with fewer CPU cores though.
